framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -196,6 +200,8 @@ def __init__(
     ):
         self.api_name = api_name
         self.api_class_prefix = api_class_prefix
+        self.camel_guard = camel_guard
```

Comment:
Is `camel_guard` necessary?  It's the same as `api_class_prefix`.

Reply:
Not always.